### PR TITLE
Use version 1.5 of the ResponsiveVoice API

### LINKED ISSUE #36: Upgrade ResponsiveVoice.JS to latest version
Currently, 1.5.7:

* https://responsivevoice.org/change-log/
* http://code.responsivevoice.org/1.5.7/responsivevoice.js

### PR DESCRIPTION
**Pull request by rodfersou**
closes #36

### CODE REVIEW

**Review by hvelarde — APPROVED** [review]
LGTM

File: src/collective/texttospeech/upgrades/v5/__init__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from collective.texttospeech.logger import logger
+from plone import api
+
+
+NEW_JS = '//code.responsivevoice.org/1.5/responsivevoice.js'
+OLD_JS = '//code.responsivevoice.org/1.4/responsivevoice.js'
+
+
+def pin_responsivevoice(setup_tool):
```
Comment (hvelarde) [inline src/collective/texttospeech/upgrades/v5/__init__.py]: seems this upgrade step is not supported under Plone 5.1; better ask @thet for advice on how to fix this.

**Review by hvelarde — CHANGES_REQUESTED** [review]

File: src/collective/texttospeech/upgrades/v5/__init__.py (hunk 1 above)
Comment (rodfersou) [reply src/collective/texttospeech/upgrades/v5/__init__.py]: @thet any advice on how to fix JS registration on Plone 5.1?

Comment (hvelarde) [reply src/collective/texttospeech/upgrades/v5/__init__.py]: take a look at https://github.com/collective/collective.lazysizes/pull/36 but I think is better to provide different profiles for different Plone versions.

**Review by hvelarde — APPROVED** [review]
LGTM

**Pull request merged**